Indicates that granting access to timezones is no longuer necessary (#108)
glpi-project/doc-install: source/timezones.rst

Grant access
------------

> .. note::
> 
>     Before GLPI 10.0.20, granting access to the ``mysql.time_zone_name`` table was required to support timezones usage. This is no longuer required.
> 

Reviewer: ```suggestion
.. versionremoved:: 10.0.20 
    Granting access to the ``mysql.time_zone_name`` is no longer required to support timezones usage.

```
